projects/angular-year-calendar/src/lib/pipes/week-number/week-number.pipe.ts

```diff
@@ -1,19 +1,34 @@
-import { Pipe, PipeTransform } from '@angular/core';
-import { differenceInCalendarDays, subDays, addWeeks, addDays, addYears, subYears, differenceInWeeks } from 'date-fns';
-import { DEFAULT_WEEK } from '../../constants/default-config';
-import { YCConfig } from '../../year-calendar-interfaces';
+import { Pipe, PipeTransform } from "@angular/core";
+import {
+  differenceInCalendarDays,
+  subDays,
+  addWeeks,
+  addDays,
+  addYears,
+  subYears,
+  differenceInWeeks,
+} from "date-fns";
+import { DEFAULT_WEEK } from "../../constants/default-config";
+import { YCConfig } from "../../year-calendar-interfaces";
 
 @Pipe({
-  name: 'weekNumber'
+  name: "weekNumber",
 })
 export class WeekNumberPipe implements PipeTransform {
-
   transform(date: Date, ycConfig: YCConfig, year): any {
     const dateClone = new Date(date);
-    const {firstWeekMonth, weekStartsOn, forceWeek, forceWeekDate} = ycConfig;
+    const oneDay = 86400000; // milliseconds in a day
```

Comment:
Can we rename this? We shouldn't need a comment to explain a variable name. The variable name should be self explanatory.

```suggestion
    const millisecondsInADay = 86400000;
```